packages/design-system/src/components/input/numericInput.js

```diff
@@ -17,82 +17,90 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import { forwardRef, useCombinedRefs } from '@googleforcreators/react';
 /**
  * Internal dependencies
  */
 import { useKeyDownEffect } from '../keyboard';
 import { useNumericInput } from './useNumericInput';
 import { Input, InputPropTypes } from '.';
 
-export const NumericInput = ({
-  allowEmpty,
-  isFloat,
-  onChange,
-  max,
-  min,
-  value = '',
-  isIndeterminate: originalIsIndeterminate,
-  ...props
-}) => {
-  const {
-    currentValue,
-    handleBlur,
-    handleChange,
-    handleEsc,
-    handleKeyUpAndDown,
-    inputRef,
-    isIndeterminate,
-  } = useNumericInput({
-    allowEmpty,
-    isFloat,
-    onChange,
-    max,
-    min,
-    value,
-    isIndeterminate: originalIsIndeterminate,
-  });
-
-  useKeyDownEffect(
-    inputRef,
+export const NumericInput = forwardRef(
```

Comment:
A big diff - but added `forwardRef` and combined both refs using the `useCombinedRefs` hook.